src/transformers/models/auto/modeling_auto.py

```diff
@@ -559,8 +559,6 @@
         ("funnel", "FunnelForQuestionAnswering"),
         ("gptj", "GPTJForQuestionAnswering"),
         ("ibert", "IBertForQuestionAnswering"),
-        ("layoutlmv2", "LayoutLMv2ForQuestionAnswering"),
-        ("layoutlmv3", "LayoutLMv3ForQuestionAnswering"),
```

Comment:
Not sure whether we can just remove those here for backwards compatibility, cc @sgugger.

This removes the ability to do:

```
from transformers import AutoModelForQuestionAnswering

model = AutoModelForQuestionAnswering.from_pretrained("microsoft/layoutlmv3-base")
```

Reply:
Yes, they should be left here to avoid a breaking change.

Reply:
Thanks for the clarification. I'll add them back in.